project/Dependencies.scala

```diff
@@ -73,6 +73,7 @@ object Dependencies {
   val playJson = "com.typesafe.play" %% "play-json" % playJsonVersion
   val playJsonJoda = "com.typesafe.play" %% "play-json-joda" % playJsonVersion
   val playIteratees = "com.typesafe.play" %% "play-iteratees" % "2.6.1"
+  val ophanEventModel = "com.gu" %% "ophan-event-model" % "1.0.0"
```

Comment:
This would be to try and use the Thrift types to lock down enums for tracking

Reply:
is this used in this PR?

Reply:
not yet. I'm debating whether it's worth it - it would be to try and enforce that some of the strings like `"ACQUISITIONS_HEADER"` are actually allowed enum values

Reply:
it is now!